benchmarks/lasso/solvers/cd.py

```diff
@@ -31,7 +29,8 @@ class Solver(BaseSolver):
     requirements = ['numba', 'scipy']
 
     def set_objective(self, X, y, lmbd):
-        self.X, self.y, self.lmbd = X, y, lmbd
+        # use Fortran order to compute gradient on contiguous columns
+        self.X, self.y, self.lmbd = np.ascontiguousarray(X.T).T, y, lmbd
```

Comment:
```suggestion
        self.X, self.y, self.lmbd = np.asfortranarray(X), y, lmbd
```

same remark for below